virttest/migration.py

```diff
@@ -36,6 +36,12 @@ def __init__(self):
         # The CmdResult returned from virsh migrate command
         self.ret = None
 
+    def _reinit(self):
```

Comment:
I could not see the requirement for us to create this new function. Could you give a detailed explanation or example?